prow/cluster/starter.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
+data:
+  plugins:
+---
```

Comment:
Should we assume that people know that `---` starts a new resource?

Reply:
Yes.